lib/test.coffee

```diff
@@ -80,6 +80,9 @@ class Test
     assert.isNull error
     assert.isNotNull response, 'Response'
 
+    # Headers
+    @response.headers = response.headers
```

Comment:
The assert on master:L84 should be verifying the expected response status (from RAML) matches
the actual response status.
Unsure why master:L89 is needed at all from a quick glance, but cannot research that right now.